moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -115,14 +115,20 @@ const ProjectView = () => {
                   aria-label="Project Details Tabs"
                 >
                   {TABS.map((tab, i) => {
-                    return <Tab label={tab.label} {...a11yProps(i)} />;
+                    return (
+                      <Tab
+                        key={tab.label}
+                        label={tab.label}
+                        {...a11yProps(i)}
+                      />
+                    );
                   })}
                 </Tabs>
               </AppBar>
               {TABS.map((tab, i) => {
                 const TabComponent = tab.Component;
                 return (
-                  <TabPanel value={activeTab} index={i}>
+                  <TabPanel key={tab.label} value={activeTab} index={i}>
```

Comment:
Thanks for doing this! I bet this reduces the console warnings significantly.

Reply:
For sure! Trying to chip away those warnings bit by bit to reduce the noise.